clang/lib/Basic/Targets/BPF.cpp

```diff
@@ -38,16 +38,22 @@ void BPFTargetInfo::getTargetDefines(const LangOptions &Opts,
 
   Builder.defineMacro("__BPF_FEATURE_ADDR_SPACE_CAST");
 
-  if (CPU.empty() || CPU == "generic" || CPU == "v1") {
+  if (CPU == "generic" || CPU == "v1") {
```

Comment:
Nitpick: I think it would be more clear to add a line like below:

```
if (CPU.empty)
  CPU = "v3";
```

(and do the same thing in `BPFSubtarget::initSubtargetFeatures`)

Reply:
Indeed. This is simpler and easy to reason. Will make the change.

Reply:
Thank you